### PR TITLE
Update SVG Icons in `@bolt/icons` Component; Add Workaround Temp Build

### PR DESCRIPTION
**Pull request by sghoweri**
- Adds missing icons from v0.x
- Update schema to use values component expects (based on .js file names auto-generated)
- Add temporary build process so icons can be regenerated; also adds tiny opt-in sub-watch process for re-building as things change
- Docs page now references values from icon schema
- Fixes a whole ton of SVG icons that were previously broken; still a few left that aren't fully working but most are shaping up!

CC @remydenton @mikemai2awesome @joekarasek @rockymountainhigh1943

### CODE REVIEW

**Comment by sghoweri** [comment]
> 1. Can you briefly describe exactly how they were broken?

image

Just some examples of how we need to continue to refine our SVG build process (the same one used in v0.x) -- mostly relating to fill colors either needing to get stripped out or inverted, but all a few places where a shape inside the SVG (like the "solid" icon options) needs to be subtracted from the icon itself vs just being one color overlayed on top of another.

TLDR: not a new issue - I previously opened a ticket to work with @MattNagy on addressing a bunch of these. The bigger thing here is that we're now getting the ability to see all the icons in the system so this QC process is more straightforward and automated: https://5a85985d7b6ee84947962828--bolt-design-system.netlify.com/?p=viewall-components-icon

> 2. Does that mean none of the broken ones were being used on the existing site, or could they have been used in a potentially broken way?

Nope - if they are broken here they've been broken in Drupal since they were first added - it just means these haven't been used (yet)

> 3. Do we need to go back and update the SVG icons in Box (and/or discuss with UX about the process for generating icons)?

For the solid icons, probably - everything else is basically just going to require us to add some additional checks and logic to the pre-build process we run all our icons through.
